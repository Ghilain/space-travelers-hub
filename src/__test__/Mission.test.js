import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../redux/ConfigureStore';
import Missions from '../pages/Mission';

const mission = [
  {
    description: 'Description example',
    id: 1,
    name: 'First mission',
    reserved: true,
  },
];

describe('testing the rendering of missions', () => {
  it('', () => {
    const { container } = render(
      <Provider store={store}>
        <Missions key={mission[0]} mission={mission[0]} />
      </Provider>,
    );
    expect(container).toMatchSnapshot();
  });
});
