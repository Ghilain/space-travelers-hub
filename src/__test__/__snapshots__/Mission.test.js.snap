// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing the rendering of missions  1`] = `
<div>
  <section>
    <table
      class="table"
    >
      <thead>
        <tr>
          <th>
            Mission
          </th>
          <th>
            Description
          </th>
          <th>
            Action
          </th>
          <th>
            Â 
          </th>
        </tr>
      </thead>
      <tbody />
    </table>
  </section>
</div>
`;
